project/home/views.py

```diff
@@ -76,24 +78,85 @@ def next_month(d):
 
 
 def event(request, event_id=None):
-
+    instance = Event()
     if event_id:
         instance = get_object_or_404(Event, pk=event_id)
-        if not request.user.has_perm('view_event', instance):
-            messages.error(request, "You do not have permission to edit this event.")
-            return HttpResponse(status=204)
-            
-    else:
-        instance = Event()
-    
```

Comment:
Why were these removed

Reply:
My bad! Thank you for catching the issues! 
Those issues happened after I tried to resolve merge conflicts
I accidentally removed it to fix my errors. Then, I just remembered that I had to reverse back before pushing!

Reply:
Because I added the recurring event, we should not pass the Event() to the instance.

Reply:
Lol. I found out that because you updated sprint3 first before I'm done with my recurring event. That's when the issue occurred.